lib/urbanopt/reopt/reopt_ghp_api.rb

```diff
@@ -0,0 +1,188 @@
+# *********************************************************************************
+# URBANopt (tm), Copyright (c) Alliance for Sustainable Energy, LLC.
+# See also https://github.com/urbanopt/urbanopt-reopt-gem/blob/develop/LICENSE.md
+# *********************************************************************************
+
+require 'net/https'
+require 'openssl'
+require 'uri'
+require 'json'
+require 'securerandom'
+require 'certified'
+require_relative '../../../developer_nrel_key'
+require 'urbanopt/reopt/reopt_logger'
+
+module URBANopt # :nodoc:
+  module REopt  # :nodoc:
+    class REoptLiteGHPAPI
+
+        def initialize(reopt_input_file, nrel_developer_key = nil, reopt_output_file, use_localhost)
+
+            # Store developer key
+            if [nil, '', '<insert your key here>'].include? nrel_developer_key
+                if [nil, '', '<insert your key here>'].include? DEVELOPER_NREL_KEY
+                    raise 'A developer.nrel.gov API key is required. Please see https://developer.nrel.gov/signup/ then update the file urbanopt-reopt-gem/developer_nrel_key.rb'
+                else
+                    #Store the NREL developer key
+                    nrel_developer_key = DEVELOPER_NREL_KEY
+                end
+            end
+
+            @use_localhost = use_localhost
+            if @use_localhost
+                @root_url = "http://localhost:8000/stable"
+            else
+                @root_url = "https://developer.nrel.gov/api/reopt/stable"
```

Comment:
We should explicitly version our calls to reopt. That means replacing `stable` with `v3` in all the urls. I'm scarred from the last version snafu we had with them.